requests/__init__.py

```diff
@@ -56,7 +56,7 @@
 # See https://github.com/kennethreitz/requests/issues/3213 as well.
 import ssl
 
-if ssl.OPENSSL_VERSION_INFO < (1, 0, 1):
+if hasattr(ssl, 'OPENSSL_VERSION_INFO') and getattr(ssl, 'OPENSSL_VERSION_INFO') < (1, 0, 1):
```

Comment:
@drpoggi so, a few things:

1. No one is calling you dense
2. No one is talking about dropping support for `requests[security]` on Python 2.6
3. This is a provisional pull request. More investigation is necessary prior to merging it.

   Specifically, we need to know what "slow" means to @dsully and figure out how to reproduce that slowness to see if we can fix it.

Finally, I don't think you intended it* but your tone in your comments is a bit hostile. Please take a step back and come back with a less adversarial approach. Your contributions have been valuable in the past, and I have reason to believe that with a more constructive and collaborative tone they'll continue to be now and in the future. 🎉 

*intentions aren't magical - they don't fix anything.